setup.py

```diff
@@ -28,7 +28,7 @@ def read_requirements(file):
     name="cishouseholds",
     version="1.3.0-beta.13",
     author="CIS development team",
-    author_email="cis.dev@ons.gov.uk",
+    author_email_address="cis.dev@ons.gov.uk",
```

Comment:
This shouldn't have been changed
```suggestion
    author_email="cis.dev@ons.gov.uk",
```